libs/community/langchain_community/tools/tavily_search/tool.py

```diff
@@ -29,6 +29,19 @@ class TavilySearchResults(BaseTool):
     )
     api_wrapper: TavilySearchAPIWrapper = Field(default_factory=TavilySearchAPIWrapper)  # type: ignore[arg-type]
     max_results: int = 5
+    """Max search results to return, default is 5"""
+    search_depth: str = "advanced"
+    '''The depth of the search. It can be "basic" or "advanced"'''
+    include_domains: List[str] = []
```

Comment:
This is missing an Optional as a possible value. 

 `limit_to_domains` would be a clearer name probably. I assume that `[]` and a `None` have entirely opposite behaviors here, so the None value should be made explicit in the type signature

Reply:
what's this have to do with callbacks?